test/WrappedCollateralToken.t.sol

```diff
@@ -77,15 +80,22 @@ contract WrappedCollateralTokenTest is Test {
 
         wrapped.setMaxBalance(500e18);
 
-        vm.expectRevert(IWrappedCollateralToken.ExceedsMaxBalance.selector);
+        vm.expectRevert(abi.encodeWithSelector(IPositionManagerDependent.CallerIsNotPositionManager.selector, account));
         wrapped.transfer(account2, 501e18);
+        vm.stopPrank();
+    }
 
-        // this one should pass
-        wrapped.transfer(account2, 300e18);
+    function testTransfers() public {
```

Comment:
I think we can delete this test, and check only that both revert in case of being called by other addresses than posman

Reply:
Resolved